VideoLocker/res/layout/panel_custom_action_bar.xml

```diff
@@ -18,13 +18,13 @@
         android:layout_marginLeft="11dp"
         android:layout_marginStart="11dp"
         android:orientation="vertical"
+        android:contentDescription="@string/cancel_btn"
```

Comment:
Also, on iOS, I believe that the word "button" is added by the operating system automatically, since you can specify the type of object something is and it said "button" after the names of several things I was sure didn't have "button" in their accessibility description. I don't know if Android also does this, but it seems plausible. That way most things can have their actual text as the accessibility description